src/main/java/jalau/at18/azul/view/FactoryButtonGroup.java

```diff
@@ -3,18 +3,22 @@
 import java.util.List;
 import javax.swing.JPanel;
 
+import jalau.at18.azul.CenterTileBoard;
 import jalau.at18.azul.Tile;
 //import jalau.at18.azul.Controller.TileController;
+import jalau.at18.azul.controllerazul.MoveTileController;
 
 public class FactoryButtonGroup extends JPanel {
-   // private TileController gameController;
+    private MoveTileController moveTiles;
   //  private List<Tile> tileList;
 
```

Comment:
Erase the comments if there are not going to be needed any more

Reply:
I removed lines with comments